mycore-neo4j/src/main/java/org/mycore/mcr/neo4j/frontend/cli/MCRNeo4JCommands.java

```diff
@@ -0,0 +1,155 @@
+      MCRNeo4JManager clazz
+         = MCRConfiguration2.getOrThrow(NEO4J_MANAGER_CLASS_PROPERTY, MCRConfiguration2::instantiateClass);
+      MCRObject mcrObject = MCRMetadataManager.retrieveMCRObject(MCRObjectID.getInstance(id));
+      clazz.updateNodeByMCRObject(mcrObject);
+   }
+
+   @MCRCommand(syntax = "synchronize neo4j for base {0}",
+      help = "synchronize metadata to the Neo4J database for MCRBase {0}", order = 60)
+   public static void synchronizeForBase(final String baseId) {
+      LOGGER.info("Synchronize Neo4J with metadata for MCRBase {}", baseId);
+      MCRNeo4JManager clazz
+         = MCRConfiguration2.getOrThrow(NEO4J_MANAGER_CLASS_PROPERTY, MCRConfiguration2::instantiateClass);
+      List<String> selectedObjectIds = MCRXMLMetadataManager
+         .instance().listIDsForBase(baseId);
+      for (String objectId : selectedObjectIds) {
+         MCRObject mcrObject = MCRMetadataManager.retrieveMCRObject(MCRObjectID.getInstance(objectId));
+         clazz.updateNodeByMCRObject(mcrObject);
+      }
+   }
+
+   @MCRCommand(syntax = "synchronize neo4j for type {0}",
+      help = "synchronize metadata to the Neo4J database for MCRType {0}", order = 70)
+   public static void synchronizeForType(final String type) {
+      LOGGER.info("Synchronize Neo4J with metadata for MCRType {}", type);
+      MCRNeo4JManager clazz
+         = MCRConfiguration2.getOrThrow(NEO4J_MANAGER_CLASS_PROPERTY, MCRConfiguration2::instantiateClass);
+      List<String> selectedObjectIds = MCRXMLMetadataManager
+         .instance().listIDsOfType(type);
+      for (String objectId : selectedObjectIds) {
+         MCRObject mcrObject = MCRMetadataManager.retrieveMCRObject(MCRObjectID.getInstance(objectId));
+         clazz.updateNodeByMCRObject(mcrObject);
+      }
+   }
+
+   @MCRCommand(syntax = "synchronize neo4j metadata", help = "synchronize all metadata the Neo4J database", order = 80)
+   public static void synchronizeAll() {
+      LOGGER.info("Synchronisation of all metadata with the Neo4J database");
+      MCRNeo4JManager clazz
+         = MCRConfiguration2.getOrThrow(NEO4J_MANAGER_CLASS_PROPERTY, MCRConfiguration2::instantiateClass);
+      List<String> selectedObjectIds = MCRXMLMetadataManager
+         .instance()
+         .listIDs();
+      for (String objectId : selectedObjectIds) {
+         MCRObject mcrObject = MCRMetadataManager.retrieveMCRObject(MCRObjectID.getInstance(objectId));
+         clazz.updateNodeByMCRObject(mcrObject);
+      }
+   }
+
+   @MCRCommand(syntax = "test neo4j connection", help = "test connectivity with the Neo4j database", order = 90)
+   public static void testConnection() {
+      LOGGER.info("Test connection to Neo4j Database");
+      MCRNeo4JDatabaseDriver.getInstance().getDriver();
+      boolean connected = MCRNeo4JDatabaseDriver.getInstance().testConnectionSettings();
+      LOGGER.info("Neo4j connected {}", connected);
+   }
+
+   @MCRCommand(syntax = "test neo4j of id {0}",
```

Comment:
```suggestion
   @MCRCommand(syntax = "test neo4j for id {0}",
```
All commands use "for" instead of "of" now except for this test command, is this intended?